cmd/evm/runner.go

```diff
@@ -27,6 +27,8 @@ import (
 
 	goruntime "runtime"
 
+	"math/big"
+
```

Comment:
Please delete all newlines form the imports and run `goimports` again, it will reorder these a bit.